src/renderer/bootstrap.tsx

```diff
@@ -88,6 +90,7 @@ export async function bootstrap(App: AppComponent) {
   const filesystemStore = FilesystemProvisionerStore.createInstance();
   const themeStore = ThemeStore.createInstance();
   const hotbarStore = HotbarStore.createInstance();
+  const weblinkStore = WeblinkStore.createInstance();
 
```

Comment:
Should we do the same here, like you did in master?

Reply:
Yes, done.